playbooks/templates/standalone_parameters.yaml.j2

```diff
@@ -81,6 +81,12 @@ parameter_defaults:
   CephPoolDefaultPgNum: 8
   CephPoolDefaultSize: 1
 {% endif %}
+{% if rhsm_enabled %}
+  ContainerImageRegistryCredentials:
+    registry.redhat.io:
+      {{ rhsm_username }}: {{ rhsm_password }} 
```

Comment:
These two variables might be uninitialized.

Reply:
I made them required for now.